package/system/apk/Makefile

```diff
@@ -51,6 +51,8 @@ MESON_ARGS += \
 	-Ddocs=disabled \
 	-Dzstd=false
 
+HOST_LDFLAGS += \
```

Comment:
why?

Reply:
Because it fails without

Reply:
Why?

Reply:
Because `apk` won't be able to locate `$STAGING_DIR/host/lib/libapk.so.*` as `$STAGING_DIR/host/lib` is not part of `LD_LIBRARY_PATH`.

Reply:
second commit from https://github.com/openwrt/openwrt/pull/14694 is needed to fix that. That PR needs merging actually...